### PR TITLE
To review

### PR DESCRIPTION
**Pull request by benh-gds**
first draft of PaaS documents to support July launch

### CODE REVIEW

File: docs/deploying_apps/deploying_rails.md [hunk 1]
```diff
@@ -1,28 +1,44 @@
-See the official CF guide: [Getting Started Deploying Ruby on Rails Apps](http://docs.cloudfoundry.org/buildpacks/ruby/gsg-ror.html). Here are the minimal steps for a basic Rails app:
 
-1. Follow the setup instructions in [Getting Started](/overview/quick_start_guide/) and review [concepts](/overview/concepts/).  If you are just getting started learning CloudFoundry, you can use the sandbox space:
+For full details of how to deploy Ruby on Rails apps, see the official Cloud Foundry guide [Getting Started Deploying Ruby on Rails Apps](http://docs.cloudfoundry.org/buildpacks/ruby/gsg-ror.html). 
 
-    ```
-    cf target -o sandbox
-    ```
+Note that the only database service currently supported by PaaS is PostgreSQL. If your Rails app requires a database, it must be able to work with PostgreSQL (this should be the case if the app follows the 12-factor principle of treating the database as a loosely-coupled backing service).
```
Comment (Jonty) [inline docs/deploying_apps/deploying_rails.md]: I don't think this is quite right. 12 factor apps must be able to work with a database in any location, supplied by any provider, but it's fine for it to require that database is MySQL - 12 factor doesn't specify that databases should comply to the same standard, or that apps should use adapter layers to be compatible with any database.

File: docs/deploying_apps/deploying_static_sites.md [hunk 2]
```diff
@@ -1,82 +1,50 @@
...
+    cf push
+    ``
 
-Create a `.travis.yml` file with `edge` set to `true`:
+The site should now be live at `https://APPNAME.cloudapps.digital`.
 
-```yml
-edge: true
-```
+For a production site, you should run at least two instances of the app to improve availability.
```
Comment (Jonty) [inline docs/deploying_apps/deploying_static_sites.md]: This should probably say something like "ensure availability" - it does improve it, but if you don't run at least two you _will_ experience downtime at some point.

File: docs/deploying_services/index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Your application probably relies on backing services such as a database, an email delivery service or a monitoring system.
+
+In Cloud Foundry, backing services are referred to as 'services' and are available through the Cloud Foundry ``marketplace`` command.
```
Comment (Jonty) [inline docs/deploying_services/index.md]: Perhaps say `cf marketplace` for clarity?

File: docs/deploying_services/postgres.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+1. From the command line, run:  
+
+    ``cf marketplace``  
+
+    to see the available services.
+
+    You will see output like this:
+
+	
+    	service     plans      								description
```
Comment (Jonty) [inline docs/deploying_services/postgres.md]: This (and the above) line should probably be in code blocks for formatting.

File: docs/managing_apps/quotas.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+	Creating app APPLICATION in org ORG / space SPACE as USER...
+	FAILED
+	Server error, status code: 400, error code: 100007, message: 
+	You have exceeded the memory limit for your organization's quota.
+
+In this situation you have three options:
+
+1. Delete existing resources with `cf delete`, `cf delete-service`, `cf delete-route` or similar commands.
+2. Reconfigure existing [application quotas](#application-quotas) and redeploy.
+3. Request a quota change: contact us at <gov-uk-paas-support@digital.cabinet-office.gov.uk>.
```
Comment (Jonty) [inline docs/managing_apps/quotas.md]: This should be a mailto: link to match the rest of the documentation.

**Comment by Jonty** [comment]
It may be worth calling out that your application might need to parse (or use a library that parses) the `VCAP_SERVICES` environment variable in order to get any bound service information.

The Ruby buildpack parses this automatically and sets `DATABASE_URL` to the first database found, but this is not true for almost all other buildpacks/languages.

File: docs/deploying_apps/deploying_static_sites.md [hunk 6]
```diff
@@ -1,82 +1,50 @@
...
+        </html>
+    
 
-If the static content is included in a different folder, you can add a `path` declaration. E.g., `path: dist` or `path: assets`.
+3. Create a `manifest.yml` which uses the [`staticfile_buildpack`](https://github.com/cloudfoundry/staticfile-buildpack):
 
-Deploy:
+        ---
+        applications:
+        + name: my-static-site
```
Comment (Rory80Hz) [inline docs/deploying_apps/deploying_static_sites.md]: This needs to be - rather than +

File: docs/deploying_apps/deploying_static_sites.md [hunk 7]
```diff
@@ -1,82 +1,50 @@
...
+        memory: 64M
+        buildpack: staticfile_buildpack
+    
+    Replace ``my-static-site`` with the name you want for your app.
```
Comment (Rory80Hz) [inline docs/deploying_apps/deploying_static_sites.md]: Might be worth nothing that if the name is not unique you may get an error message, for example:

FAILED
Server error, status code: 400, error code: 210003, message: The host is taken: healthcheck

**Pull request merged**